avm/ptn/authorization/policy-assignment/tests/e2e/rg.min/main.test.bicep

```
@@ -0,0 +1,56 @@
+targetScope = 'managementGroup'
+metadata name = 'Policy Assignments (Resource Group)'
+metadata description = 'This module deploys a Policy Assignment at a Resource Group scope using minimal parameters.'
+
+// ========== //
+// Parameters //
+// ========== //
+
+@description('Optional. The name of the resource group to deploy for testing purposes.')
+@maxLength(90)
+param resourceGroupName string = 'dep-${namePrefix}-authorization.policyassignments-${serviceShort}-rg'
+
+@description('Optional. The location to deploy resources to.')
+param resourceLocation string = deployment().location
+
+@description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints.')
+param serviceShort string = 'apargmin'
+
+@description('Optional. A token to inject into the name of each resource.')
+param namePrefix string = '#_namePrefix_#'
+
+@description('Optional. Subscription ID of the subscription to assign the RBAC role to. If no Resource Group name is provided, the module deploys at subscription level, therefore assigns the provided RBAC role to the subscription.')
+param subscriptionId string = '#_subscriptionId_#'
+
+// ============ //
+// Dependencies //
+// ============ //
+
+// General resources
+// =================
+module resourceGroupDeploy 'br/public:avm/res/resources/resource-group:0.2.3' ={
+  scope: subscription('${subscriptionId}')
+  name: '${uniqueString(deployment().name, resourceLocation)}-resourceGroup'
+  params: {
+    name: resourceGroupName
+    location: resourceLocation
+  }
+}
+
+// ============== //
+// Test Execution //
+// ============== //
+
+module testDeployment '../../../main.bicep' = {
+  name: '${uniqueString(deployment().name)}-test-${serviceShort}'
+  params: {
+    name: '${namePrefix}${serviceShort}001'
+    policyDefinitionId: '/providers/Microsoft.Authorization/policyDefinitions/06a78e20-9358-41c9-923c-fb736d382a4d'
+    metadata: {
+      assignedBy: 'Bicep'
+    }
+    location: resourceLocation
+    resourceGroupName: resourceGroupName
```

If you want to use the resourge group module, I'd recommend to add an implicit dependency to avoid errors: 
```suggestion
    resourceGroupName: resourceGroupDeploy.outputs.name
```